package.json

```diff
@@ -79,7 +79,7 @@
     "@bbc/gel-foundations": "1.2.0",
     "@bbc/psammead-assets": "^0.1.9",
     "@bbc/psammead-brand": "^1.0.1",
-    "@bbc/psammead-caption": "^1.0.1",
+    "@bbc/psammead-caption": "^1.1.3",
```

Comment:
Following merge of https://github.com/bbc/psammead/pull/524 this needs to be bumped to `^1.1.5` and snapshots need updating